src/bilder/images/consul/deploy.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+
+from pyinfra import host
+
+from bilder.components.baseline.setup import install_baseline_packages
+from bilder.components.caddy.models import CaddyConfig, CaddyPlugin
+from bilder.components.caddy.steps import caddy_service, configure_caddy, install_caddy
+from bilder.components.hashicorp.consul.models.consul import (
+    Consul,
+    ConsulACL,
+    ConsulACLToken,
+    ConsulConfig,
+    ConsulTelemetry,
+)
+from bilder.components.hashicorp.consul.steps import proxy_consul_dns
+from bilder.components.hashicorp.steps import (
+    configure_hashicorp_product,
+    install_hashicorp_products,
+    register_services,
+)
+from bilder.facts import has_systemd  # noqa: F401
+
+VERSIONS = {  # noqa: WPS407
+    "caddy_route53": "v1.1.1",
+    "consul": "1.9.5",
+}
+
+install_baseline_packages()
+consul_configuration = {
+    Path("00-default.json"): ConsulConfig(
+        acl=ConsulACL([ConsulACLToken(master="<value>")]),
+        bootstrap_expect=3,
+        encrypt="",
```

Comment:
You can leave this out as setting the environment variable will handle creating the necessary value.